keras_hub/src/models/gemma3/gemma3_backbone.py

```diff
@@ -108,10 +109,84 @@ class Gemma3Backbone(Backbone):
         'use_sliding_window_attention': True,
         'vision_encoder': None,
         'layer_norm_epsilon': 1e-06,
-        dtype: "bfloat16",
     }
 
-    model = keras_hub.models.Gemma3Backbone(**config)
+    model = keras_hub.models.Gemma3Backbone(
+        vocabulary_size=262144,
+        image_size=896,
+        num_layers=34,
+        num_query_heads=8,
+        num_key_value_heads=4,
+        hidden_dim=2560,
+        intermediate_dim=10240,
+        head_dim=256,
+        query_head_dim_normalize=True,
+        use_post_ffw_norm=True,
+        use_post_attention_norm=True,
+        final_logit_soft_cap=None,
+        attention_logit_soft_cap=None,
+        sliding_window_size=1024,
+        use_sliding_window_attention=True,
+        vision_encoder=None,
+        layer_norm_epsilon=1e-06,
+        dtype="bfloat16",
+    )
+    model(input_data)
+
+    # === Vision + Language Gemma3 model ===
+    input_data = {}
+    input_data["images"] = np.ones(shape=(1, 1, 896, 896, 3))
+    input_data["token_ids"] = np.ones(shape=(1, 300), dtype="int32")
+    # images after the text part of the sequence.
+    input_data["vision_mask"] = np.expand_dims(
+        np.array([0] * 30 + [1] * 256 + [0] * 14),
+        axis=0,
+    ).astype(bool)
+    input_data["vision_indices"] = (
+        np.expand_dims(np.arange(30, 286), axis=0)
+    )
+    input_data["padding_mask"] = (
+        np.expand_dims(np.array([1] * 286 + [0] * (300 - 286)), axis=0)
+        .astype(bool)
+    )
+
+    # Pretrained Gemma3 decoder.
+    model = keras_hub.models.Gemma3Backbone.from_preset("gemma3_instruct_4b")
+    model(input_data)
+
+    vision_encoder = Gemma3VisionEncoder(
```

Comment:
remove this right? this is the pretrained gemma section?

Reply:
Don't we show both in doc-strings?
https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/bert/bert_backbone.py#L66C5-L66C62